README.md

```diff
@@ -2,18 +2,6 @@
 
 Unofficial native OpenRepos.net client for Sailfish OS
 
-[![Release](https://img.shields.io/github/release/mentaljam/harbour-storeman.svg)]()
-
-[![](https://openrepos.net/sites/default/files/styles/medium/public/packages/6416/screenshot-screenshot-storeman-01.png)](https://openrepos.net/sites/default/files/packages/6416/screenshot-screenshot-storeman-01.png)
```

Comment:
For some reason screenshots become available under another links. They should be fixed but not removed.

Reply:
In line with [my previous comment](https://github.com/mentaljam/harbour-storeman/pull/73/files#r236048826) on this PR:
* Obviously the links to the screenshots need manual maintenance every now and then (at least checking that they are working).
* The sreenshots are also available elsewhere (on OpenRepos, hence linking there, below).
* Screenshots are not necessary on a developer site (as Github): People coming here will usually have some experience with Storeman (at least installed and opened it).

Reply:
Hence my suggestion WRT the screenshots is also to just omit them to be freed of that maintenance burden.